### PR TITLE
Unifies unit tests filename suffix to .test.js

### PR DESCRIPTION
**Pull request by tmashuang**
Unifies the unit tests to a `.test.js` filename extension suffix.

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]
Excellent, I love this @tmashuang

**Comment by Gudahtt** [comment]
I tried looking for files with `test` in the name, and found a few missing from this PR:

```
/test/unit/app/controllers/transactions/tx-helper-test.js
./test/unit/app/controllers/transactions/tx-gas-util-test.js
./test/unit/app/controllers/transactions/pending-tx-tracker-test.js
./test/unit/app/controllers/transactions/tx-state-history-helpers-test.js
./test/unit/app/controllers/transactions/tx-utils-test.js
./test/unit/app/controllers/transactions/tx-state-manager-test.js
./test/unit/app/controllers/transactions/tx-controller-test.js
```

I also found a few remaining with the `spec.js` suffix:

```
/test/unit/ui/app/actions.spec.js
./test/unit/ui/app/reducers/metamask.spec.js
./test/unit/ui/app/reducers/app.spec.js
./test/unit/ui/etherscan-prefix-for-network.spec.js
```

**Comment by Gudahtt** [comment]
Oh, found one more!

`./test/unit-global/frozenPromise.js`

That's all I can find though. I just finished manually scanning the test directory.

**Review by Gudahtt — DISMISSED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -20,10 +20,10 @@
     "forwarder": "node ./development/static-server.js ./node_modules/@metamask/forwarder/dist/ --port 9010",
     "dapp-forwarder": "concurrently -k -n forwarder,dapp -p '[{time}][{name}]' 'yarn forwarder' 'yarn dapp'",
     "sendwithprivatedapp": "node development/static-server.js test/e2e/send-eth-with-private-key-test --port 8080",
-    "test:unit": "mocha --exit --require test/env.js --require test/setup.js --recursive \"test/unit/**/*.js\" \"ui/app/**/*.test.js\"",
+    "test:unit": "mocha --exit --require test/env.js --require test/setup.js --recursive \"test/unit/**/*.test.js\" \"ui/app/**/*.test.js\"",
     "test:unit:global": "mocha --exit --require test/env.js --require test/setup.js --recursive test/unit-global/*",
```
Comment (Gudahtt) [inline package.json:24]: Nit: We could consider making this more specific now as well:

```suggestion
    "test:unit:global": "mocha --exit --require test/env.js --require test/setup.js --recursive test/unit-global/*.test.js",
```

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**